sync/clients/sync.py

```diff
@@ -109,7 +110,16 @@ def delete_project(self, project_id: str) -> dict:
         return self._send(self._client.build_request("DELETE", f"/v1/projects/{project_id}"))
 
     def _send(self, request: httpx.Request) -> dict:
-        response = self._client.send(request)
+        try:
+            for attempt in Retrying(stop=stop_after_attempt(3), wait=wait_fixed(5)):
+                with attempt:
+                    response = self._client.send(request)
+                    if response.status_code == httpx.codes.SERVICE_UNAVAILABLE:
+                        # Doesn't need to be any particular exception, this is just to trigger the retry
+                        raise Exception()
```

Comment:
Looks like `tenacity.TryAgain` was made to be used in places like this.

Reply:
ah, nice find!